tests/test_frame.py

```diff
@@ -172,6 +172,9 @@ def test_types_to_csv() -> None:
     # Testing support for binary file handles, added in 1.2.0 https://pandas.pydata.org/docs/whatsnew/v1.2.0.html
     df.to_csv(io.BytesIO(), quoting=csv.QUOTE_ALL, encoding="utf-8", compression="gzip")
 
+    with ensure_clean() as path:
```

Comment:
you'll have to test the python version here as well

Reply:
done